BlackCubes/covidmap-react-native
# Searchbar Component

## Changes
1. Created a component called `Searchbar` so that users could enter a location on the map.

## Purpose
There needs to be a searchbar for users to enter a location or find one for the map.

## Approach
In order for the user to search for a location on the map, a `Searchbar` component was created. When the user enters inside the input, an inner border appears but it would disappear once the user exits out the component with no inputs. Yet, if there was an input and the user exits out the component, the inner border would still appear. This was done by utilizing the `onFocus` and `onBlur` props.

## Testing
1. Pull changes.
2. On the terminal, go to the `/covid-mapper` directory, and run `npm install` if there were any packages not installed.
3. Then, run `expo start` and select your emulator.
4. Once your emulator runs with expo, make sure the searchbar appears.
5. Make sure the inner searchbar border appears when click on the searchbar.
6. Make sure the inner searchbar border disappears when you click outside with no inputs.
7. Make sure the inner searchbar border still appears when you click outside with inputs.

## Screenshots

https://user-images.githubusercontent.com/29642735/137800437-d1b54070-1afe-47f2-a6e8-4e0d91fb0359.mp4



Closes #5
